sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosBridgeInternal.java

```diff
@@ -32,6 +32,11 @@ public static AsyncDocumentClient getAsyncDocumentClient(CosmosClient client) {
         return client.asyncClient().getDocClientWrapper();
     }
 
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static CosmosAsyncClient getCosmosAsyncClient(CosmosClient client) {
```

Comment:
we should avoid adding more BridgeInternal APIs as per discussion with the central SDK team.

You can rely on the pattern that we have here:
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ImplementationBridgeHelpers.java

Reply:
done, created CosmosClientAccessor